letsencrypt/storage.py

```diff
@@ -252,6 +252,7 @@ def __init__(self, config_filename, cli_config):
         self.privkey = self.configuration["privkey"]
         self.chain = self.configuration["chain"]
         self.fullchain = self.configuration["fullchain"]
+        self.live_dir = os.path.dirname(self.cert)
```

Comment:
We'll have a lot of thinking to do about what this means if we end up reorganizing the cert storage so that the four PEM objects are not necessarily in the same directory (as I believe djb is going to advocate for security/privilege separation reasons). I agree that this construction of `live_dir` is totally correct in our status quo.